### DISCUSSION TITLE
Issue with addon not starting correctly on HA restart (DEV Version)

### DISCUSSION THREAD

**Discussion by snigehere** [tsightler/ring-mqtt#471] [Support Q&A]
Ring-MQTT with Video Streaming Current version: 5.0.3  (Note development version)
Mosquitto broker Current version: 6.1.3
Home Assistant 2022.9.7
Supervisor 2022.09.1
Operating System 9.0
Frontend 20220907.2 - latest

I notices that since the last but 1 update of HA the ring-mqtt addon is not correctly starting when HA is restarted.  Most entities show up as "unavailable" after the HA restart.  This can be corrected by stopping and starting the ring mqtt add on... checking mqtt explorer I see just a few entities registered but in the add on log I see multiple being processed... 

I am running the dev version following trying to solve other issues but as it did not help I could switch back to the last stable release if needed.

Logs and screen capture of mqtt explorer are available if it would be helpful.

**Comment by tsightler** [comment]
> I am running the dev version following trying to solve other issues but as it did not help I could switch back to the last stable release if needed.

Dev version currently has a known issue detecting HA restarts and republishing as there's some refactor going on in that code with the ultimate goal of supporting dynamic device add/remove.  It may have been fixed with some updates that I pushed last night, but the dev code is in a particularly unstable state right now.

**Comment by snigehere** [comment]
OK - Thanks - I will switch back to latest
